Assets/MixedRealityToolkit/UX/Scripts/BoundingBoxes/BoundingBoxGizmoShell.cs

```diff
@@ -9,7 +9,7 @@ namespace MixedRealityToolkit.UX.BoundingBoxes
     /// Draws a bounding box gizmo in the style of the hololens shell
     /// </summary>
     [ExecuteInEditMode]
-    public class BoundingBoxGizmoShell : BoundingBoxGizmo
+    public class BoundingBoxBasic : BoundingBoxGizmo
```

Comment:
We should probably use an obsolete marking in an empty class to tell devs what the new class name is.

Reply:
i changed the type of the variable BoundingBoxGizmo. The name of the class didn't change.

Reply:
This class name change was unnecessary. Reverted this change.